.github/workflows/pr-request-review.yml

```diff
@@ -0,0 +1,42 @@
+name: Request Review
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  request-review:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    if: github.repository_owner == 'cryostatio' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/request_review')
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        
+      - name: Add label and request review
+        run: |
+          reviewers="@cryostatio/reviewers"
+
+          echo "Adding label 'review-requested' to the PR"
+          gh pr edit ${{ github.event.issue.number }} --add-label "review-requested"
+
+          if [ -n "$reviewers" ]; then
+            gh pr request-review ${{ github.event.issue.number }} --reviewer $reviewers
+          else
+            echo "Requesting review from @cryostatio/reviewer(s) team"
+            gh pr request-review ${{ github.event.issue.number }} --team cryostatio/reviewer(s)
+          fi
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Remove label on review
```

Comment:
This should be triggered under https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_review

We can extract it into another workflow file.

Reply:
the remove label after review?

Reply:
trying to figure out why the adding is failing first.

Reply:
> the remove label after review?

Yes!!